components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/utils/mappings/APIMappingUtil.java

```diff
@@ -969,13 +972,24 @@ public static LifecycleHistoryDTO fromLifecycleHistoryModelToDTO(List<LifeCycleE
 
                 uriTemplates.add(template);
             } else {
-                handleException("The HTTP method '" + httpVerb + "' provided for resource '" + uriTempVal
-                        + "' is invalid");
+
+                if(model.getType().equals(APIConstants.GRAPHQL_API)){
```

Comment:
Formatting issue and swap equals condition.

Reply:
Fixed